packages/lodestar/test/e2e/network/peers/peerManager.test.ts

```diff
@@ -122,6 +122,9 @@ describe("network / peers / PeerManager", function () {
     // We get a ping by peer1, don't have it's metadata so it gets requested
     networkEventBus.emit(NetworkEvent.reqRespRequest, {method: ReqRespMethod.Ping, body: seqNumber}, peerId1);
 
+    // Allow to set metadata in onPing
+    await sleep(0);
```

Comment:
Why is this necessary? Could you clarify "Allow to set metadata in onPing"?

Reply:
method `onPing` of PeerManager becomes async so we need this line to make sure it complete and we can confirm `reqResp.metadata.callCount` below